src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -185,79 +201,293 @@ sub-01_run-2_T1w.json
 ```
 
 The `run-<index>` in the example above denotes the index of the acquisition 
-repeated with the identical parameters (e.g. to achieve a higher SNR). Please 
+repeated with the **identical parameters **(e.g. to achieve a higher SNR). Please 
 note that changing parameters between multiple acquisitions of the same sequence
-creates a different use case: parametrically linked anatomical images.
+creates a different use case: `parametrically linked anatomical images`.
+
+The entries listed for `suffixes for conventional MRI contrasts` in the list of 
+available suffixes are immutable. Descriptions of this type of suffixes are preceded 
```

Comment:
Can we rephrase this to say something like:

modality_label must be chosen from the designated list below. The list of modality labels can only be adjusted by extending the bids specification.